crates/database/src/ethflow_orders.rs

```diff
@@ -130,7 +130,12 @@ AND o.partially_fillable = false
 AND t.order_uid is null
 AND eo.valid_to < $1
 AND o.sell_amount = oq.sell_amount
-AND (1.0 - o.buy_amount / oq.buy_amount) >= $3
+AND (
+    CASE
+        WHEN oq.buy_amount = 0 THEN FALSE
```

Comment:
I think we should not discard these orders. After all the `sell_amount` might not be 0 meaning the user actually has some `ETH` locked in our contract which should be refunded at some point in time, no?

Reply:
Makes sense yes, but I don't understand this whole condition with slippage and `FALSE` seemed like a safe default.
This condition discards all orders where `o.buy_amount == oq.buy_amount` which does not make sense to me.

Reply:
I'm tempted to even drop this whole slippage condition.

Reply:
According to the slippage condition we don't refund order with `buy_amount` higher than market `buy_amount` (some kind of limit order right), but why wouldn't we if it is already expired.

Reply:
> I don't understand this whole condition with slippage

The initial idea was the we would only like to refund orders that got created with a reasonable slippage (the UI only allows to place orders with such slippage). We didn't want to "waste" money on orders that were too unlikely to be filled. I.e. if the user sets the slippage too low it's their own fault so they can pay the cost of calling the `refund` function on the smart contract.

> I'm tempted to even drop this whole slippage condition.

I think this would be fine as all the refunder pods are currently running with `minSlippageBps: 0` which effectively disables that condition anyway.